.vscode/recommended-settings.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
good to have `  "files.eol": "\n",` option too.

Reply:
we have this already in .editorconfig ```end_of_line = lf```